src/AspNet.Security.OAuth.Introspection/OAuthIntrospectionHandler.cs

```diff
@@ -148,7 +149,7 @@ public class OAuthIntrospectionHandler : AuthenticationHandler<OAuthIntrospectio
         protected virtual Task<bool> ValidateAudienceAsync(JObject payload) {
             // If no explicit audience has been configured,
             // skip the default audience validation.
-            if (string.IsNullOrEmpty(Options.Audience)) {
+            if (Options.Audiences == null || Options.Audiences.Count <= 0) {
```

Comment:
Not sure I really like settable collection properties :smile: 

nit: `IList.Count` cannot be negative.